pkg/unikontainers/config.go

```diff
@@ -36,6 +36,9 @@ type UnikernelConfig struct {
 	UnikernelBinary string `json:"com.urunc.unikernel.binary"`
 	Hypervisor      string `json:"com.urunc.unikernel.hypervisor"`
 	Initrd          string `json:"com.urunc.unikernel.initrd,omitempty"`
+	Block           string `json:"com.urunc.unikernel.block,omitempty"`
+	BlkMntPoint     string `json:"com.urunc.unikernel.blkMntPoint,omitempty"`
+	UseDMBlock      string `json:"com.urunc.unikernel.useDMBlock"`
```

Comment:
Yes, that would be more appropriate. However, `Map` in unikernelConfig returns a map of strings and therefore, we would need to convert it back to string again. Later, when we want to use it we will need to convert it from string to bool. Therefore, I thought we could just keep it as string to avoid converting it multiple times.